packages/eslint-plugin-template/tests/configs.test.ts

```diff
@@ -1,15 +1,18 @@
 import eslintPluginTemplate from '../src';
 
+const ESLINT_PLUGIN_PREFFIX = '@angular-eslint/template/';
+
 interface Config {
   extends?: string | string[];
   rules?: { [ruleName: string]: string | object };
   overrides?: Config[];
 }
 
 function containsRule(config: any, ruleName: string): boolean {
+  const prefixedRuleName = `${ESLINT_PLUGIN_PREFFIX}${ruleName}`;
   return (
-    Boolean(config.rules?.[ruleName]) ||
-    config.overrides?.some((config: Config) => config.rules?.[ruleName])
+    Boolean(config.rules?.[prefixedRuleName]) ||
+    config.overrides?.some((config: Config) => config.rules?.[prefixedRuleName])
```

Comment:
To avoid a possible mental confusion with `config` received in the parameter, could you rename it as follows?

```suggestion
    config.overrides?.some(({ rules }) => rules?.[prefixedRuleName])
```